docs/maintenance/release-management.md

```diff
@@ -57,6 +57,9 @@ In the beginning of each iteration we start by releasing new development version
 - invenio-communities [GitHub](https://github.com/inveniosoftware/invenio-communities)
     - bump invenio-requests
     - bump invenio-vocabularies
+- invenio-administration [GitHub](https://github.com/inveniosoftware/invenio-administration)
```

Comment:
it should be higher in the order, before vocabularies

Reply:
hmmm I'm leaning towards the opposite (maybe we are understanding the list in a diff direction), vocabularies should be released before in case it needs to be bumped. See: https://github.com/inveniosoftware/invenio-administration/blob/main/setup.cfg#L35.